templates/can_msg_defs.mako.h

```diff
@@ -18,7 +18,12 @@ typedef enum {
   ${helpers.generate_enum(can_messages, 'CAN_MESSAGE')}
 } CanMessage;
 
-<% can_frames = parse_can_frames(options.filename) %> 
+% for _, dev_name in can_devices.items():
+#define CAN_MSG_ACK_DEVICE_${dev_name} 1 << CAN_DEVICE_${dev_name}
```

Comment:
wrap in brackets please :)

Looking at the generated file, it might get a little confusing dealing with so many macros. Maybe a macro function to transform device IDs into their expected ACK bitset forms would be easier to understand?

Reply:
Will do in next PR... Oops.

Reply:
I made a variadic macro that does the transformation :)
Will be coming in my CAN ACK bitset PR